The serial jenkins tester is unhappy about a mismatch in output:
```
3052/4764 Test #3175: lac/householder.debug .................................................................***Failed    9.93 sec

***  End of file "/jenkins/workspace/dealii-serial_PR-18469/tests/lac/householder.output" reached while trying to read line 3.
***  File "output" has more lines than file "/jenkins/workspace/dealii-serial_PR-18469/tests/lac/householder.output",
***  line 3 is the last one read from file "output"

make[3]: *** [CMakeFiles/lac.householder.debug.test.dir/build.make:78: householder.debug/serial/diff] Error 1
make[2]: *** [CMakeFiles/Makefile2:1854: CMakeFiles/lac.householder.debug.test.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:1861: CMakeFiles/lac.householder.debug.test.dir/rule] Error 2
make: *** [Makefile:797: lac.householder.debug.test] Error 2
Test lac/householder.debug: DIFF
===============================   OUTPUT BEGIN  ===============================
[100%] Building CXX object CMakeFiles/lac.householder.debug.dir/householder.cc.o
[100%] Linking CXX executable householder.debug/householder.debug
[100%] Built target lac.householder.debug
[100%] Normalizing test output file /home/dealii/build/tests/lac/householder.debug/serial/output
[100%] Generating householder.debug/serial/diff
lac/householder.debug: BUILD successful.
lac/householder.debug: RUN successful.
lac/householder.debug: DIFF failed. ------ Source: /jenkins/workspace/dealii-serial_PR-18469/tests/lac/householder.output
lac/householder.debug: DIFF failed. ------ Result: /home/dealii/build/tests/lac/householder.debug/serial/output
Check /home/dealii/build/tests/lac/householder.debug/serial/output /jenkins/workspace/dealii-serial_PR-18469/tests/lac/householder.output
lac/householder.debug: DIFF failed. ------ Diff:   /home/dealii/build/tests/lac/householder.debug/serial/failing_diff
lac/householder.debug: DIFF failed. ------ First 20 lines of numdiff output:
----------------
          <==
##3       ==> DEAL::Distance (1.75,0.00)


+++  File "/jenkins/workspace/dealii-serial_PR-18469/tests/lac/householder.output" differs from file "output"


lac/householder.debug: ******    DIFF failed    *******
```

--- a/include/deal.II/lac/householder.h
+++ b/include/deal.II/lac/householder.h
@@ -19,7 +19,6 @@
 #include <deal.II/base/config.h>
 
 #include <deal.II/lac/full_matrix.h>
-#include <deal.II/lac/vector_memory.h>
 
 #include <cmath>
 #include <vector>

--- a/tests/lac/householder.cc
+++ b/tests/lac/householder.cc
@@ -16,6 +16,7 @@
 // Tests Householder class for QR-decomposition
 
 #include <deal.II/lac/full_matrix.h>
+#include <deal.II/lac/full_matrix.templates.h>
 #include <deal.II/lac/householder.h>
 #include <deal.II/lac/vector.h>
 
@@ -36,11 +37,29 @@ main()
   FullMatrix<double>  A(4, 3, &rect[0][0]);
   Householder<double> H(A);
 
-  Vector<double> u(4);
-  Vector<double> v1(3);
-  Vector<double> v2(3);
+  {
+    Vector<double> u(4);
+    Vector<double> v1(3);
 
-  for (unsigned int i = 0; i < u.size(); ++i)
-    u(i) = i * i;
-  deallog << "Distance " << H.least_squares(v1, u) << std::endl;
+    for (unsigned int i = 0; i < u.size(); ++i)
+      u(i) = i * i;
+    deallog << "Distance " << H.least_squares(v1, u) << std::endl;
+  }
+
+  // make a rudimentary complex test
+  FullMatrix<std::complex<double>> B(A.m(), A.n());
+  for (unsigned int i = 0; i < A.m(); ++i)
+    for (unsigned int j = 0; j < A.n(); ++j)
+      B(i, j) = A(i, j);
+
+  Householder<std::complex<double>> HB(B);
+
+  {
+    Vector<std::complex<double>> u(4);
+    Vector<std::complex<double>> v1(3);
+
+    for (unsigned int i = 0; i < u.size(); ++i)
+      u(i) = i * i;
+    deallog << "Distance " << H.least_squares(v1, u) << std::endl;
+  }
 }

--- a/tests/lac/householder.output
+++ b/tests/lac/householder.output
@@ -1,2 +1,3 @@
 
 DEAL::Distance 1.75
+DEAL::Distance (1.75,0.00)

